### ISSUE TITLE
RTX Super Resolution support for videos

### ISSUE THREAD

**Issue by revanmj** [brave/brave-browser#28303]
## Description 
Will Brave support feature mentioned in this article (as all news about this so far mention only Chrome and Edge)

https://www.ghacks.net/2023/02/06/chrome-and-edge-support-nvidia-rtx-super-resolution/

**Comment by rishiraj88** [comment]
Thanks a lot, @revanmj , for the link.

**Comment by UjCbFwtBayFM** [comment]
Unless Brave decided it was best to manually cut out that feature, it should already be available on the pre-release stable branch. Though Nvidia has not yet released the drivers to actually enable it.

**Comment by ForthtuN** [comment]
Really hope Brave will have that feature. Chrome got it the other day in their stable branch. It will need to be implemented,

**Comment by UjCbFwtBayFM** [comment]
They don't need to implement it.

**Comment by xkeyC** [comment]
### According to my tests, it works fine in Brave (full screen playback)

#### After enabling it, the CPU usage increases significantly, and the picture has also been enhanced to a certain extent.

#### I think Nvidia has this feature for any Chromium (110+)

![image](https://user-images.githubusercontent.com/39891083/222135656-44288175-cb27-4e0f-819e-4e92d14201d0.png)

![image](https://user-images.githubusercontent.com/39891083/222135456-e1c19d83-935c-422b-b982-689628e44914.png)

**Comment by bsclifton** [comment]
Thanks for testing @xkeyC! We've had Chromium 110 since the 1.48- we'll be shipping Chromium 111 coming next Tuesday (March 7). I'll close this issue out as it looks like we get it from upstream code

**Comment by McLovinI87** [comment]
@bsclifton 
This issue should not ne closed. the feature works fine for me in chrome and edge but will not work in brave. attached is an image of a 1080p youtube video and the control panel showing it not working
![brave-error-report](https://github.com/brave/brave-browser/assets/18416356/0d4c1848-e32b-4251-b950-5ad8b0bd8891)

**Comment by BennettStaley** [comment]
> @bsclifton This issue should not ne closed. the feature works fine for me in chrome and edge but will not work in brave. attached is an image of a 1080p youtube video and the control panel showing it not working ![brave-error-report](https://private-user-images.githubusercontent.com/18416356/320243875-0d4c1848-e32b-4251-b950-5ad8b0bd8891.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1ODkwOTEsIm5iZiI6MTcxMzU4ODc5MSwicGF0aCI6Ii8xODQxNjM1Ni8zMjAyNDM4NzUtMGQ0YzE4NDgtZTMyYi00MjUxLWI5NTAtNWFkOGIwYmQ4ODkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIwVDA0NTMxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM3MGJhMGMwOWJkOWY5YWVlYmQxOGVjNzA1N2UzMDk5ZTQ5NGNiZjA3OGVkZDM0ODA0MjQ1NGE0Mzg0OTdmNDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.NFY1DzHsbGjdaqWEgEJ2A3A_mmOGzSO1XKlqKyy3ouc)

Same on my end. RTX 4090, latest windows 11 build and brave release.

**Comment by xkeyC** [comment]
![image](https://github.com/brave/brave-browser/assets/39891083/1f905245-8e39-476c-83b4-85d6e6398c42)

It works fine for me, maybe you need a higher resolution screen and play in full screen?

```
RTX4090 Game Ready-552.22-Apr 16,2024 
Windows11 23H2 22631.3527  
Brave 1.65.126 Chromium: 124.0.6367.118
```

**Comment by aarstar** [comment]
I think this is still an issue. I am running the latest Edge (135.0.3179.73), Chrome (135.0.7049.85), and Brave (1.77.97) browsers on Windows 11 24H2 and Super Resolution works on Edge and Chrome but not Brave when watching videos on YouTube.

**Comment by emi069** [comment]
Found a working fix for this. Opened #58774 with the full details and measurements.

**TL;DR:** `brave://flags` → **Skia Graphite** → **Enabled** → Relaunch. (Or `brave.exe --enable-features=SkiaGraphite`.)

Default Brave runs:

```
Skia Backend  : GaneshGL
Skia Graphite : Disabled
```

Switching Skia to the Graphite backend is what makes VSR attach to the video. On my RTX 3060 / Win10 / Brave 1.94.121, GPU power during playback went **31–33 W → 68.7 W** and the `RTX VSR` indicator appeared (Edge reference on the same machine: 110 W).

Tip if you are testing this: turn on NVIDIA App → System → Video → Super Resolution → *Show status indicator in video* first. Otherwise you are judging by eye, which is unreliable.

Not needed: `--no-sandbox`, a driver downgrade, or `--use-angle=d3d9` — that option no longer exists in the ANGLE backend picker on Chromium 152.

Caveat: Graphite is experimental and changes Brave's whole rasterisation path, so it is a workaround, not a fix.

@McLovinI87 @BennettStaley @aarstar — does this reproduce on your 4090 / 3080 setups?
